manifests/v/Viber/Viber/15.5.0.0/Viber.Viber.locale.en-US.yaml

```diff
@@ -1,8 +1,10 @@
+# Created using YamlCreate.ps1
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
 PackageIdentifier: Viber.Viber
-PackageVersion: 15.4.1.1
+PackageVersion: 15.5.0.0
 PackageLocale: en-US
-Publisher: Viber Media S.a.r.l
+Publisher: 2010-2021 Viber Media S.a.r.l
```

Comment:
```suggestion
Publisher: Viber Media S.a.r.l
```
It shows up for me as:
![image](https://user-images.githubusercontent.com/11600822/122282063-e67fd480-cee2-11eb-99db-bea0e3b11ba7.png)

Reply:
wat, is it the MSI or EXE? I got suprised when i noticed the exe had `2010-2021 Viber Media S.a.r.l` as publisher

Reply:
It was the `.msi` installer which winget picked as default when I installed it.

Reply:
The MSI uses the ProductCode to match so its okay, or we need split them again

Reply:
There might actually be a issue where x86 is being picked over neutral, it installed the exe once i swapped the exe to x86

```
Found Viber [Viber.Viber]
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://download.cdn.viber.com/desktop/windows/ViberSetup.exe
  ██████████████████████████████   102 MB /  102 MB
Successfully verified installer hash
Starting package install...
Successfully installed
```

![image](https://user-images.githubusercontent.com/15158490/122352173-27adce00-cf4f-11eb-86e8-5a00d5743165.png)

However it somehow did find a match
```
PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs> winget list
Name                  Id                                           Version          Available   Source
------------------------------------------------------------------------------------------------------
Viber                 Viber.Viber                                  15.5.0.0
```